### DISCUSSION TITLE
Using Payment sheet how to add user email and user name for storing customer details

### DISCUSSION THREAD

**Discussion by iamravindra** [flutter-stripe/flutter_stripe#852] [Q&A]
Hi Team I am using payment sheet method for doing stripe payment but how to i would submit customer details to the stripe payment

**Comment by remonh87** [comment]
You need to create a Stripe customer on the backend and associate the customerId + ephermal key  in the paymentsheet.  By doing this you link the payment to the customer. For more details see: https://stripe.com/docs/api/customers 

Hope this helps!

**Reply by iamravindra** [reply]
Thanks remonh87 for your advice
